doc/source/tutorial/stats/sampling_srou.rst

```diff
@@ -0,0 +1,90 @@
+.. _sampling-srou:
+
+Simple Ratio-of-Uniforms (SROU)
+===============================
+
+.. currentmodule:: scipy.stats.sampling
+
+* Required: PDF, area under PDF
+* Optional: mode
+* Speed:
+
+  * Set-up: fast
+  * Sampling: slow
+
+SROU is based on the ratio-of-uniforms method that uses universal inequalities for constructing
+a (universal) bounding rectangle. It works for T-concave distributions with T(x) = -1/sqrt(x).
```

Comment:
Either render as math or code the equation.

Reply:
done